src/components/Categories.js

```diff
@@ -1,4 +1,6 @@
+import React from 'react';
+
 const Categories = () => <p>Under construction</p>;
-  <button type="button">Check Status</button>;
+  <button type="button">Check Progress</button>;
```

Comment:
The `categories component` should contain:

* A button for checking the status of the books
  * this button would use a function `checkCategoriesStatus` (name can be different) made in the component (the name may be different)
    * `checkCategoriesStatus` dispatches the `underConstruction` action creator (name of the function can be different) with the id as its argument

Kindly implement the above project requirement.

Reply:
Okay, managed to fix this one too! This was harder!